libnavigator/proguard-rules.pro

```diff
@@ -0,0 +1,27 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
+
+-keep class com.mapbox.navigation.navigator.MapboxNativeNavigatorImpl { *; }
+
+# --- Navigator ---
+-keep class com.mapbox.navigator.** { *; }
+-keep class com.mapbox.bindgen.** { *; }
```

Comment:
This should be handled downstream in NN, we shouldn't add these rule (it's not harmful though) cc @mskurydin @SiarheiFedartsou

Reply:
need this rules in child, sure.  Now they aren't exist, so as a solution till rules appear at navigator it works

Reply:
Of course! @mskurydin @SiarheiFedartsou could you cut a ticket so these are included downstream in NN when you have a chance?

Reply:
Went ahead and created a follow up ticket in NN